app/views/guests/show.html.erb

```diff
@@ -0,0 +1,17 @@
+<h2>Guest Name: <%= @guest.name %></h2>
+
+<h3>Rooms:</h3>
+<ul id="room_info">
+<% @rooms.each do |room| %>
+    <li><strong><%= room.suite %></strong>: <%= number_to_currency(room.rate) %>, <%= room.hotel.name %></li><br>
+<% end %>
+</ul>
+
+<h3>Add a Room:</h3>
+<div id="add_room">
+  <%= form_with url: "/guests/#{@guest.id}/guest_rooms", method: :post, local: true do |form| %>
+    <%= form.label :room_id, "ID:" %>
+    <%= form.number_field :room_id %>
+    <%= form.submit :Submit %>
+  <% end %>
+</div>
```

Comment:
I already mentioned this in the `routes.rb` file, but the URL for the form isn't correct according to REST.

```ruby
<%= form_with url: "/guests/#{@guest.id}/rooms", method: :post, local: true do |form| %>
```